yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -94,30 +105,52 @@ export class L2Block {
     numPublicCallsPerTx = 3,
     numEncryptedLogsPerCall = 2,
     numUnencryptedLogsPerCall = 1,
-    numL1ToL2MessagesPerCall = 2,
+    inHash: Buffer | undefined = undefined,
   ): L2Block {
     const body = Body.random(
       txsPerBlock,
       numPrivateCallsPerTx,
       numPublicCallsPerTx,
       numEncryptedLogsPerCall,
       numUnencryptedLogsPerCall,
-      numL1ToL2MessagesPerCall,
     );
 
     const txsEffectsHash = body.getTxsEffectsHash();
 
-    return L2Block.fromFields({
-      archive: makeAppendOnlyTreeSnapshot(1),
-      header: makeHeader(0, l2BlockNum, txsEffectsHash),
-      body,
-    });
+    return L2Block.fromFields(
+      {
+        archive: makeAppendOnlyTreeSnapshot(1),
+        header: makeHeader(0, l2BlockNum, txsEffectsHash, inHash),
+        body,
+      },
+      // just for testing purposes, each random L2 block got emitted in the equivalent L1 block
+      BigInt(l2BlockNum),
```

Comment:
Reverting this accidental change https://github.com/AztecProtocol/aztec-packages/pull/5371.